### DISCUSSION TITLE
duckdb how IGNORECASE

### DISCUSSION THREAD

**Discussion by jxccc2009** [duckdb/duckdb#5678] [Q&A]
Dears, 
how duckdb can  IGNORECASE while use con.execute(...).df() while under python enviroment?
as you know , if use pandas/sqldf then can query result IGNORECASE , but duckdb  seems strictly, 
so , is there anywhere to IGNORECASE souce text,
thanks.

**Comment by Mause** [comment]
Can you please give an example of a query that isn't behaving as you expect?

**Reply by jxccc2009** [reply]
for example, there's a table named 'sample' and likes below:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/j00645328/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/j00645328/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link=blue vlink=purple>



C_BG | value
-- | --
Other | a
other | b



</body>

</html>

if use below sql_query by sqldf, then can get all  no matter other or Other:

from pandasql import sqldf
sqldf( "select * from sample where C_BG like ‘%other%’ ")

but if use above same sql_query by duckdb, then can gen only the last row of table sample
import duckdb
con=duckdb.connect()
con.execute(" select * from sample where C_BG like  ‘%other%’ ").df()
